src/main/java/org/zalando/planb/revocation/api/exception/AncientRevocationException.java

```diff
@@ -0,0 +1,16 @@
+package org.zalando.planb.revocation.api.exception;
+
+/**
+ * Thrown when a client tries to post a revocation with an {@code issued_before} timestamp in the ancient past.
+ *
+ */
+public class AncientRevocationException extends RuntimeException {
+
+    private static final long serialVersionUID = 1L;
+
+    private static final String MESSAGE = "'issued_before' cannot be set in the ancient past.";
```

Comment:
Didn't like the phrasing. Also I would put more context (If we have it). Would write:

> 'issued_before' too old. Value must be greater than <timestamp here>

Where <timestamp here> is now - maxTimeDelta